packages/create-svelte/templates/default/.gitignore

```diff
@@ -1,12 +1,18 @@
-.DS_Store
+.output
+.vercel
+/package
 node_modules
-/build
+
+# Output
 /.svelte-kit
-/package
-.env
```

Comment:
Continuing the conversation from https://github.com/sveltejs/kit/pull/11881#discussion_r1501619871

Are people ever going to want to check in `.env`? It's probably good that we're updating this so that `.env.test` is not ignored, but perhaps we could at least continue ignoring `.env` since I think some maintainers were previously worried about accidentally checking that in.

I'm not sure whether it's better to add `!.env.test` or remove `.env.*`. I guess it depends what people would expect to have happen to something like `.env.qa`. I feel like that probably shouldn't be checked in either. Perhaps it'd be better to try to get Vite to switch their templates to be:
```
.env.*
!.env.example
!.env.test
```

Reply:
I personally check-in `.env` file in a Vite project, following the Vite convention.

Though I do believe that the following Vite convention is not intuitive:

```
.env                # loaded in all cases
.env.local          # loaded in all cases, ignored by git
.env.[mode]         # only loaded in specified mode
.env.[mode].local   # only loaded in specified mode, ignored by git
```

since in most Node.js projects, `.env` is usually the secret file.

> Perhaps it'd be better to try to get Vite to switch their templates

I would love this to happen - I just want to align Vite and SvelteKit ignore files.

Reply:
@hyunbinseo I think we need to leave `.env` here because the maintainers were quite worried about accidentally checking in secrets